### PR TITLE
Rollup rating on Scaling View

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-680

Note: This is only dark mode design

### CODE REVIEW

File: packages/frontend/src/components/table/RatingCell.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+export function RatingCell({ item }: RatingProps) {
+  if (!item) {
+    return <NoInfoCell />
+  }
+
+  return (
+    <div
+      className="Tooltip"
+      title={renderToStaticMarkup(<RatingTooltipPopup item={item} />)}
```
Comment (sz-piotr) [inline packages/frontend/src/components/table/RatingCell.tsx:16]: Interesting idea.

Alternative suggestion: Maybe we can instead have something like `data-tooltip-content="some-id"` and then we have this content hidden as a child of `RatingCell` and we use this `.clone()` trick from earlier to replace tooltip content?

**Review by sz-piotr — CHANGES_REQUESTED** [review]

**Comment by sz-piotr** [comment]
Also on firefox the bullet points are misaligned
Screenshot 2023-02-06 at 11 54 26

File: packages/frontend/src/components/rating/Badge.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    case 'A':
+      return 'bg-green-500 text-black'
+    case 'B':
+      return 'bg-yellow-100 text-black'
+    case 'C':
+      return 'bg-orange text-black'
+    case 'D':
+      return 'bg-red-400 text-white'
+    case '-':
+      return 'bg-red-900 text-white'
```
[the remark this answers is not in the document]
Comment (michalsidzej) [reply packages/frontend/src/components/rating/Badge.tsx:39]: I don't think so. This is related to the projects that are not rollups, so have no rating. We spoke about that and decided it should probably look like something worse than`D` category.

File: packages/frontend/src/components/rating/Tooltip.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+export function RatingTooltip() {
+  const tooltipRef = useRef<HTMLSpanElement>(null)
+  useEffect(() => {
+    configureTooltips()
+    // show tooltip
+    tooltipRef.current?.dispatchEvent(new MouseEvent('mouseenter'))
+    // remove all event listeners, leaving them will cause reg-viz to remove tooltip
+    document.body.replaceWith(document.body.cloneNode(true))
+    tooltipRef.current?.replaceWith(tooltipRef.current.cloneNode(true))
```
[the remark this answers is not in the document]
Comment (michalsidzej) [reply packages/frontend/src/components/rating/Tooltip.stories.tsx:21]: I literaly just copied that from `Tooltip.stories.tsx`

**Comment by michalsidzej** [comment]
> Also on firefox the bullet points are misaligned Screenshot 2023-02-06 at 11 54 26

Nice one, haven't checked that

**Comment by michalsidzej** [comment]
@sz-piotr Should we go for browser native approach to <ul> marker? That's how it looks on chrome
Screenshot 2023-02-06 at 12 48 00

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**